resources/views/dashboard/admin/roster/edit.blade.php

```diff
@@ -7,452 +7,341 @@
-                            1 => 'MTR',
-                            2 => 'INS'
-                        ], $user->train_position, ['class' => 'form-control', 'disabled']) !!}
+                        {!! Form::label('ctr', 'Center Certification') !!}
+                        {!! Form::select('ctr', [
+                        0 => 'None',
+                        99 => 'Solo Certification',
+                        1 => 'Certified'
+                        ], $user->ctr, ['class' => 'form-control', $center_disable]) !!}
                     </div>
-                    @if($user->hasRole('mtr') || $user->hasRole('ins'))
-                        <div class="col-sm-6">
-                            {!! Form::label('max', 'Training Level') !!}
-                            {!! Form::select('max', [
-                                null => 'Not Able to Train',
-                                1 => 'Minor/ DEL & GMD',
-                                2 => 'Major/ DEL & GMD',
-                                3 => 'Minor Local',
-                                4 => 'Major Local',
-                                5 => 'Minor Approach',
-                                6 => 'Major Approach',
-                                7 => 'Center'
-                            ], $user->max, ['class' => 'form-control', 'disabled']) !!}
-                        </div>
-                    @endif
-                @endif
+                </div>
             </div>
-        </div>
-        @if(Auth::user()->isAbleTo('roster') || Auth::user()->isAbleTo('train'))
             <div class="form-group">
                 <div class="row">
                     <div class="col-sm-6">
-						{!! Form::label('twr_solo_fields', 'Class D Tower Solo Certifications') !!}
-						{!! Form::text('twr_solo_fields', $user->twr_solo_fields, ['class' => 'form-control','maxlength' => 255]) !!}
+                        {!! Form::label('twr_solo_fields', 'Unrestricted Solo Certifications (list facility IDs)') !!}
+                        {!! Form::text('twr_solo_fields', $user->twr_solo_fields, ['class' => 'form-control','maxlength' => 255, $train_disable]) !!}
                     </div>
                     <div class="col-sm-6">
-                    {!! Form::label('twr_solo_expires', 'Tower Solo Expiration Date', ['class' => 'form-label']) !!}
-                    <div class="input-group date" id="datetimepicker1" data-target-input="nearest">
-                        {!! Form::text('twr_solo_expires', $user->twr_solo_expires, ['placeholder' => 'MM/DD/YYYY', 'class' => 'form-control datetimepicker-input', 'data-target' => '#datetimepicker1']) !!}
-                        <div class="input-group-append" data-target="#datetimepicker1" data-toggle="datetimepicker">
-                            <div class="input-group-text"><i class="fa fa-calendar"></i></div>
+                        {!! Form::label('twr_solo_expires', 'Solo Expiration Date', ['class' => 'form-label']) !!}
```

Comment:
Do we do any validation on this? Cannot be >90d per GCAP without permission from the division iirc.

Reply:
Waiting to hear back from TA on this. Most of our solo certs are defined as 30-days in the training order. Will fix when I hear back with his intent

Reply:
TA confirms - 30 days for all solo certs. I'll hard-code this